### DISCUSSION TITLE
Alignments rate

### DISCUSSION THREAD

**Discussion by seongguheo** [milaboratory/mixcr#1178] [Q&A] [answered]
Dear Authors,

The below plots is generated from a smart-seq2 single cell RNA-seq data set.
Where does the rate coming from? 
Does it from the total RNA-seq read counts?

What could be the reason for the row success rate?

![alignQc_total](https://github.com/milaboratory/mixcr/assets/130780185/bbbc43c9-73d3-487f-9854-b8150770e314)

Thank you
Luke

**Comment by Alex-Davydov** [comment, marked as answer]
Hi,
The 'Alignments rate' shows the fraction of aligned reads out of total sequencing reads. In the context of RNA-seq data (including smart-seq2 single cell data), the proportion of reads aligned to IG/TCR genes is typically low and varies based on the specific cell type. Therefore, your image appears suitable for representing this particular type of data.

**Reply by seongguheo** [reply]
Hi Alex,

Thank you for your kind reply.
Please have a look at my mixcr run code and the clonal change usage plot of the above data.
It showed that there were IGH usage as well, but I have got only a very few (less than 1/10) of IGH CDR3 sequence from the cells.

The cells were sampled from bone marrow or blood of multiple myeloma.
The fastq is paired end with 36 read length each of the pair.

---<mixcr run code>-----
mixcr analyze -Xmx32g -f smartseq2-vdj-full-length -s hsa -t 4 \
../../input/NB501311_0918/{{CELL0ROW:a}}_R1_001.fastq.gz \
../../input/NB501311_0918/{{CELL0ROW:a}}_R2_001.fastq.gz \
../../output/mixcr_june0723/mixcr_2

---<plot>-----
![chainUsage_total](https://github.com/milaboratory/mixcr/assets/130780185/1fe8886c-2640-441a-bf9e-4b968c07f32a)

Many thanks,
Luke

**Comment by mizraelson** [comment]
Hi, according to the plot, IGH clones (represented by orange) contribute to 10-20% in every sample. So, it makes sense that only 10-20% of your clones are IGH. For this type of protocol, it is normal to see uneven chain distribution.
